rs/replicated_state/src/canister_state/system_state.rs

```diff
@@ -1557,6 +1650,14 @@ impl SystemState {
             _ => None,
         }
     }
+
+    pub fn set_on_low_wasm_memory_hook_status(&mut self, status: OnLowWasmMemoryHookStatus) {
+        self.on_low_wasm_memory_hook_status = status;
+    }
+
+    pub fn get_on_low_wasm_memory_hook_status(&self) -> &OnLowWasmMemoryHookStatus {
+        &self.on_low_wasm_memory_hook_status
+    }
```

Comment:
That is explained in PR description with:

"[EXC-1724](https://dfinity.atlassian.net/browse/EXC-1724) In which we will refactor SystemState::task_queue to be new struct with additional field on_low_wasm_memory_hook_status, to ensure that whenever available first task that will be poped form task queue is OnLowWasmMemoryHook (excluding paused executions). For that reason SystemState::on_low_wasm_memory_hook status is private, and we use set and get, so it can be easier encapsulated in the future TaskQueue struct."